components/raftstore/src/store/fsm/store.rs

```diff
@@ -1186,10 +1186,7 @@ impl RaftBatchSystem {
         box_try!(workers
             .consistency_check_worker
             .start(consistency_check_runner));
-
-        if let Err(e) = sys_util::thread::set_priority(sys_util::HIGH_PRI) {
-            warn!("set thread priority for raftstore failed"; "error" => ?e);
-        }
+        let _ = sys_util::thread::set_priority(sys_util::HIGH_PRI);
```

Comment:
It's not running in raftstore threads.

Reply:
The description updated

Reply:
It's wrong because we wanted to set priority for raftstore threads but  in fact we set it in main thread.

Reply:
You probably want to remove it or move into batch system, specially at .https://github.com/tikv/tikv/blob/ed71f20f445e10595553d2bf3d1a1eb645b9a61a/components/batch-system/src/batch.rs#L394.

Reply:
done